### DISCUSSION TITLE
[FEATURE] Save only locked weapons and artifacts

### DISCUSSION THREAD

**Discussion by LepusCavum** [Andrewthe13th/Inventory_Kamera#334] [Ideas or Feature requests]
## What feature are you proposing?
It would be helpful to include the ability to only scan artifacts that are locked so as to weed out trash artifacts you do not intend to keep anyways

**Comment by reloxx13** [comment]
if u do it urself why u even use the scanner or optimizer

**Comment by LepusCavum** [comment]
> if u do it urself why u even use the scanner or optimizer

Currently you can choose what rarity and what level of artifacts you'd like to scan. I'd like to scan only locked artifacts so I can use the resulting JSON for another program. Artifacts I leave unlocked I use to leveling things up, and currently I have to manually go though the JSON and delete the unlocked artifacts, so that I can only compare the artifacts I actually want to keep and consider.

**Comment by Cupcak3** [comment]
If you are using the artifact list generated for any other tool other than Genshin Optimizer, let me know. Otherwise Genshin Optimizer has a "Delete Artifacts" feature that removes all artifacts that are not locked, regardless if it equipped on a character, from the database. I recommend using that feature as it sounds like what you are asking for.

**Comment by LepusCavum** [comment]
Interesting, I will check on the Genshin Optimizer. Currently I use the program here: https://genshin.aspirine.su/

Apologies for the close and reopen: misclick.
